setup.py

```diff
@@ -21,7 +21,7 @@
     ],
     packages=find_packages('.'),
     install_requires=[
-        'safety',
+        'safety>=2.0.0',
```

Comment:
It seems preferable to at least pin the major version number if not the minor and patch version numbers as well so that this doesn't happen again with the release of safety 3.x.

Reply:
Yep, going to add this pin

Reply:
Updated, thanks